cicd/cmd/run-build/main.go

```diff
@@ -35,7 +35,8 @@ func main() {
 		mvnFlags.SkipShade(),
 		mvnFlags.SkipSpotlessCheck(),
 		mvnFlags.SkipTests(),
-		mvnFlags.ThreadCount(3))
+		mvnFlags.ThreadCount(3),
+		mvnFlags.InternalMaven())
```

Comment:
Is this accessible to everyone? If not, should we disable it for non-CI/release builds by default?

Reply:
@damccorm All builds on GitHub actions have access to the Airlock repo, and the `cicd/` folder only runs on GitHub actions, so there shouldn't be a case where these don't run, unless I'm missing something

Reply:
> and the cicd/ folder only runs on GitHub actions

You're right, this is the part I wasn't considering - I forgot people don't use these for their normal local development. Honestly, that workflow seems wrong to me (ideally there would be little divergence from remote/local to make it easy to repro issues), but that's out of scope here